packages/whale-api-client/src/api/oracle.ts

```diff
@@ -0,0 +1,36 @@
+import { WhaleApiClient } from '../whale.api.client'
+
+/**
+ * DeFi whale endpoint for oracle related services.
+ */
+export class Oracle {
+  constructor (private readonly client: WhaleApiClient) {
+  }
+
+  /**
+   * Get current status of an oracle
+   *
+   * @param {string} id oracleId
+   * @return {Promise<OracleStatus>}
+   */
+  async getStatus (id: string): Promise<OracleStatus> {
+    return await this.client.requestData('GET', `oracle/${id}/status`)
+  }
+}
+
+export interface OracleStatus {
```

Comment:
```suggestion
export interface OracleAppointed {
```

I think it's better to call it appointed since that's the action